app/views/edition_changes/show.html.erb

```diff
@@ -17,7 +17,7 @@
         <br/>
         <small>
           Changes between this draft and
-          <%= link_to_if @edition.previously_published_edition, "the previous version",  @edition.previously_published_edition %>.
+          <%= link_to_if @edition.previously_published_edition, "the previous version",  edition_changes_path(@edition.previously_published_edition) %>.
```

Comment:
It's a sentence explaining what the user currently sees, so keep it as:
> Changes between this draft and the previous version.

But without linkifying the last part